config/samples/agent-install.openshift.io_v1beta1_agentserviceconfig.yaml

```diff
@@ -1,8 +1,7 @@
----
 apiVersion: agent-install.openshift.io/v1beta1
 kind: AgentServiceConfig
 metadata:
- name: agent
+  name: agent
```

Comment:
Why changing this?

Reply:
I believe the original was spaced incorrectly. Whatever the reason, It got updated by the little `python -c` and I don't think it's worth fighting.